### PR TITLE
bugfix_19795

### PR DESCRIPTION
**Pull request by xus**
[http://www.ilias.de/mantis/view.php?id=19795](http://www.ilias.de/mantis/view.php?id=19795)

### CODE REVIEW

File: setup/sql/dbupdate_custom.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	$storage = new ilFSStorageExercise($row['exc_id'], $row['id']);
+
+	//if the assignment has criteria category
+	if($row['peer_crit_cat'])
+	{
+		//get criteria id which is uploader
+		$res_crit = $db->query("SELECT id FROM exc_crit ".
+			" WHERE parent = ".$row['peer_crit_cat'].
+			" AND type = 'file'"
+		);
```
Comment (alex40724) [inline setup/sql/dbupdate_custom.php:51]: This is missing proper quoting

File: setup/sql/dbupdate_custom.php [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+		'order_nr' => array(
+			'type' => 'integer',
+			'length' => 4
+		)
+	);
+
+	$ilDB->createTable("exc_ass_file_order", $fields);
+	$ilDB->addPrimaryKey("exc_ass_file_order", array("id"));
+	$ilDB->createSequence("exc_ass_file_order", 1);
+}
```
Comment (alex40724) [inline setup/sql/dbupdate_custom.php:29]: This belongs to the new feature development. I guess I can safely ignore this when merging.

Comment (xus) [reply setup/sql/dbupdate_custom.php:29]: Yes, it belongs to the new feature, you can ignore it.

File: setup/sql/dbupdate_custom.php [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+		$res_crit = $ilDB->query("SELECT id FROM exc_crit ".
+			" WHERE parent = ".$ilDB->quote($row['peer_crit_cat'],"integer").
+			" AND type = ".$ilDB->quote('file','string')
+		);
+
+		while($row_crit = $ilDB->fetchAssoc($res_crit))
+		{
+			$original_path = $storage->getPeerReviewUploadPath($row['peer_id'], $row['giver_id'], $row_crit['id']);
+
+			$path_peaces = explode('/', rtrim($a_original_path, '/'));
```
Comment (alex40724) [inline setup/sql/dbupdate_custom.php:57]: This does not seem to be defined.

File: setup/sql/dbupdate_custom.php [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+				}
+			}
+		}
+	}
+	//assignment without criteria (just rename the file and move it to the proper directory)
+	else
+	{
+		$original_path = $storage->getPeerReviewUploadPath($row['peer_id'], $row['giver_id'], $row['peer_crit_cat']);
+
+		$path_peaces = explode('/', rtrim($a_original_path, '/'));
```
Comment (alex40724) [inline setup/sql/dbupdate_custom.php:109]: again, see above

**Comment by alex40724** [comment]
Thanks xus. For others: The merge looks very different from these commits (no custom update merged, e.g.).

**Pull request merged**